README.md

```diff
@@ -143,6 +144,22 @@ If you’re using another framework or package manager that doesn’t support NP
 
 If you’re interested in maintaining a package that helps us distribute the Draft U.S. Web Design Standards, the project’s build system can help you create distribution bundles to use in your project. Please read our [contributing guidelines](CONTRIBUTING.md#building-the-project-locally-with--gulp-) to locally build distributions for your framework or package manager.
 
+### Use the latest version of the `uswds` package
+
+If you'd like to run the latest version of the Standards, follow these steps:
+
+1. Clone the latest version of the [web-design-standards repo](https://github.com/18F/web-design-standards/tree/staging).
```

Comment:
Since this is the `web-design-standards` repo, you shouldn't need a link to it (you're here!). We should probably link to the `web-design-standards-docs` repo instead.